packages/CollapsibleCard/package.json

```diff
@@ -15,25 +15,28 @@
   },
   "dependencies": {
     "@babel/runtime-corejs2": "^7.3.1",
-    "@paprika/avatar": "^2.2.4-alpha.1",
-    "@paprika/collapsible": "^1.1.5",
+    "@paprika/button": "^1.0.6",
+    "@paprika/counter": "^1.0.4",
+    "@paprika/heading": "^1.0.6",
     "@paprika/helpers": "^2.1.1-alpha.1",
+    "@paprika/icon": "^1.0.4",
     "@paprika/stylers": "^1.0.0",
     "@paprika/tokens": "^1.0.0",
     "prop-types": "^15.7.2",
     "uuid": "^8.3.2"
   },
+  "devDependencies": {
+    "@paprika/avatar": "^2.2.2",
+    "@paprika/checkbox": "^1.0.5",
+    "@paprika/overflow-menu": "^1.0.9",
+    "@paprika/status-tracker": "^1.0.6",
+    "@paprika/switch": "^1.0.5"
+  },
   "peerDependencies": {
     "@paprika/l10n": "^1.1.3",
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
     "styled-components": "^4.3.1"
   },
-  "paprikaDocs": {
-    "subComponents": [
-      "Avatar",
-      "Metadata"
-    ]
-  },
```

Comment:
It was my understanding that this `paprikaDocs` section will include these subcomponents in the auto-generated documentation. I didn't see a lot of value in that, as most of them just take `children` as a prop (though two do have some other props). I figure people will look at the stories for guidance.

What do you think?

Reply:
That works too if you don't need the auto-generated documentation. However you'll need to update this doc https://github.com/acl-services/paprika/blob/169b7e6142c1ae71b3a57c2e9b80f216a8054446/packages/OverflowMenu/README.md manually since it's still showing the old apis

Reply:
The only difference in the OverflowMenu is a new `onClick` prop, and its documentation was updated:
https://github.com/acl-services/paprika/pull/923/files#diff-bfc200c242a99575dbc2d315547c8e6756568309038beb05003c7e8328329bbcR63
and
https://github.com/acl-services/paprika/pull/923/files#diff-b3606f85c3964c87d81275db1c4dbfa050c459d52f23c697721e964c50342b8aR67